### ISSUE TITLE
Hazardous column on waste table

### ISSUE THREAD

**Issue by e-manifest** [USEPA/e-manifest#502]
Add column to waste table to indicate if the waste is hazardous. Column text should be "HM" with a alt text that displays "Hazardous Material" when hovering over it. Display between Line Number and US DOT Description.
